src/Login/backend.ts

```diff
@@ -0,0 +1,19 @@
+import { secretCheck } from "../secret/secret"
+
+export function LoginToken() {
```

Comment:
Hier handelt es sich ja nicht um eine Komponente in dem Sinn, dass sie von React gerendert wird. Daher sollte die Funktion mit einem Verb beschrieben werden, welches dann klein oder in Zusammenhang  mit  anderen Subjeken im CamelCase geschrieben wird. Beispielsweise: aquireToken
Aber getToken oder loadToken tun es auch.
Außerdem würde ich empfehlen, email und passwort als Eingabeparameter dieser Funktion zu definieren.

Ach und bevor ich's vergesse... Wo sind die Tests? Gerade für das Mocken von HTTP-Requests gibt es jede Menge libraries. Lassen Sie sich doch von diesen Links hier inspirieren:

- [https://dev.to/softchris/http-mocking-for-your-tests-in-react-26f6](https://dev.to/softchris/http-mocking-for-your-tests-in-react-26f6)

- [https://www.chaijs.com/plugins/chai-http/](https://www.chaijs.com/plugins/chai-http/)